src/wa_crypt_tools/wadecrypt.py

```diff
@@ -215,6 +216,29 @@ def chunked_decrypt(file_hash, cipher, encrypted, decrypted, buffer_size: int =
         encrypted.close()
 
 
```

Comment:
this function is not needed as the Crypt15 class can already be printed normally

Reply:
Ok I really remembered I wrote __str__ for Db classes, it looks like it's not the case, my bad. I will do it